### ISSUE TITLE
PA#1 results

### ISSUE THREAD

**Issue by benluiwj** [nus-cs2030/2021-s1#260]
Motivation after seeing your PA#1 results

Hi everyone, today we received our PA#1 results. Im sure some of you are satisfied with the results and with that congratulations to you! For those who performed below your expectations(I did), don't give up! Sure this PA is 15% of the entire grade and it is not a small percentage, but there are still other components that have not been graded(such as the project) that still can be done to your best abilities! So continue to work hard and know that the end is insight. Your efforts in PA1 may not have yielded the results you wanted but there is still time to improve your overall grade with the project, PA#2 and of course finals! 
Press on everyone!

TLDR: there's still hope for those who dint perform well for PA1
